src/main/java/com/lb/ai/tools/EmailAPITool.java

```diff
@@ -0,0 +1,65 @@
+package com.lb.ai.tools;
+
+import jakarta.mail.MessagingException;
+import jakarta.mail.Session;
+import jakarta.mail.Transport;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import java.util.Properties;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.ai.tool.annotation.Tool;
+
+public class EmailAPITool {
+
+  private static final String smtpHost = "localhost";
+  private static final String smtpPort = "1025";
+  private static final Logger log = LoggerFactory.getLogger(EmailAPITool.class);
+  private static final AtomicBoolean sentEmail = new AtomicBoolean(false);
```

Comment:
Since this is static it means that only one email is sent. Wasn't the intention that only one email per session would be sent? I think you create the tool instance each time so removing static would be enough?

Reply:
you're very right. 
Removed.